MetaMask/metamask-extension
# Revert stake button and announcement

## **Description**

Reverts the commits related to staking. The relevant PRs being reverted are:
- https://github.com/MetaMask/metamask-extension/pull/22347
- https://github.com/MetaMask/metamask-extension/pull/22291
- https://github.com/MetaMask/metamask-extension/pull/22511

## **Related issues**

No issue for the revert exists.

## **Manual testing steps**

1. Install the build and open MM
2. Look at the "What's new" announcements
3. Confirm that the one for staking has been removed but the following one (Blockaid) remains
4. Go to token listing
5. Confirm that the "Stake" button no longer appears by the mainnet ETH token symbol

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
